OJP/OJP_Trips.xsd

```diff
@@ -722,6 +727,25 @@
 			<xs:enumeration value="upAndDown"/>
 			<xs:enumeration value="downAndUp"/>
 		</xs:restriction>
+	</xs:simpleType>
+		<xs:simpleType name="AccessibilityFeaturesTypeEnumeration">
```

Comment:
at the end of the day, aren't you expecting here something being NeTEx _AccessibilityInfoFacilityEnumeration_ + _MobilityFacilityEnumeration_ + _AssistanceFacilityEnumeration_ + _MobilityFacilityEnumeration_ ?

Reply:
Yes, at the end of the day that may be what we want / need. The intention at the beginning of the day was: being able to return (a) the information necessary to tell whether a PathLink is feasible for mobility impaired passengers (route planning) plus (b) a few more attributes that I could imagine a routing app might want to display to a mobility impaired user. (b) is somewhat arbitrary and could be extended as you propose - to be able to return more or less any accessibilty feature on the PathLink that happens to be found.

Reply:
@Aurige  @trurlurl  Can you finalise the discussion? What should be done?

Reply:
@Aurige I have elaborated on your idea to see what it would mean to add still more accessibility information. What do you think?

*AccessFeatureType* as proposed:
stairs
seriesOfStairs
singleStep
seriesOfSingleSteps
escalator
travelator
ramp
footpath
other
unknown

PLUS 
shuttle (from AccessFacility)

---

*AccessibilityFeaturesTypeEnumeration*

From MobilityFacility...
lowFloor -> lowFloorAccess (might be preferrable)
stepFreeAccess
suitableForPushchairs
suitableForWheelchairs
suitableForHeaviliyDisabled -> suitableForHeavilyDisabled (typo in NeTEx)
boardingAssistance
onboardAssistance
unaccompaniedMinorAssistance
tactilePlatformEdges
tactileGuidingStrips
raisedKerb

PLUS (seems to make sense)
elevatorWithTactileButtons
elevatorWithAcousticAnnouncements
tactileOrAuditorySigns
suitableForUnassistedWheelchairs
suitableForBicycles
visualSigns

PLUS  (from AccessFacility)
barrier


From VehicleAccessFacility,,,
wheelchairLift
manualRamp
automaticRamp
steps
slidingStep
narrowEntrance
validator


From AccessibilityInfoFacility...
audioInformation
audioForHearingImpaired
visualDisplays
displaysForVisuallyImpaired
largePrintTimetables